security-analysis/security-analysis-api/src/main/java/com/powsybl/security/monitor/StateMonitor.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.monitor;
+
+import java.util.*;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ *
+ *
+ * A stateMonitor allows to get information about branch and bus on the network after a security analysis computation.
+ * Contingency context allows to specify if the informations asked are about N situation N-k situation with a contingency id
+ * or both.
+ *
+ */
+public class StateMonitor {
+
+    /**
+     * define on which situation information are needed
+     */
+    private final ContingencyContext contingencyContext;
+
+    /**
+     * branchs ids on which information will be collected
+     */
+    private final Set<String> branchIds = new LinkedHashSet<>();
+    /**
+     * voltageLevels ids on which information will be collected
+     */
+    private final Set<String> voltageLevelIds = new LinkedHashSet<>();
```

Comment:
Do we need order in these sets? If not, you can use `HashSet`

Reply:
Yes I need them